tests/test_lens.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+
+import dynsight
+import h5py
+import numpy as np
+
+"""
```

Comment:
Move this to the doc string of the tests.

Reply:
i think i don't understand what you mean

Reply:
Move the comments from line 7 to after line 21 (make them to doc string to `test_lens_signals`). Do this for both test files.